src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors /
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+
+protocol RiskFailedCollectionViewCellDelegate: AnyObject {
+	func activeButtonTapped(cell: RiskFailedCollectionViewCell)
+}
+
+final class RiskFailedCollectionViewCell: HomeCardCollectionViewCell {
+
+	weak var delegate: RiskFailedCollectionViewCellDelegate?
+
+	@IBOutlet var titleLabel: ENALabel!
+	@IBOutlet var chevronImageView: UIImageView!
+	@IBOutlet var bodyLabel: ENALabel!
+	@IBOutlet var activeButton: ENAButton!
+
+	@IBOutlet var viewContainer: UIView!
+	@IBOutlet var topContainer: UIView!
+	@IBOutlet var stackView: UIStackView!
+	@IBOutlet var riskViewStackView: UIStackView!
```

Comment:
private ?

Reply:
Not possible. Its accessed by the configurator.